rpcclient/infrastructure.go

```diff
@@ -103,6 +105,22 @@ type jsonRequest struct {
 	responseChan   chan *response
 }
 
+// BackendVersion represents the version of the backend the client is currently
+// connected to.
+type BackendVersion uint8
+
+const (
+	// BitcoindPre19 represents a bitcoind version before 0.19.0.
+	BitcoindPre19 BackendVersion = iota
+
+	// BitcoindPost19 represents a bitcoind version equal to or greater than
+	// 0.19.0.
+	BitcoindPost19
+
+	// Btcd represents a catch-all btcd version.
+	Btcd
```

Comment:
Since we only need to differentiate `BitcoindPost19` from the other backends, seems like we could save us save some trouble by just having two types for now.

Reply:
Don't we also need to distinguish `btcd` before and after the series of changes in this PR?

Reply:
No, there aren't any changes being done at `btcd`'s RPC level.